AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/editor_script_canvas.py

```diff
@@ -62,28 +59,46 @@ def create_new_entity_with_component(self, entity_name: str, sc_file_path: str,
         self.editor_entity = EditorEntity.create_editor_entity_at(position, entity_name)
         self.editor_entity.add_components([SCRIPT_CANVAS_UI])
 
-        self.script_canvas_component = self.editor_entity.get_components_of_type([SCRIPT_CANVAS_UI])[0]
-        self.script_canvas_component.set_component_property_value(SCRIPT_CANVAS_COMPONENT_PROPERTY_PATH, sourcehandle)
+        self.script_canvas_components = self.editor_entity.get_components_of_type([SCRIPT_CANVAS_UI])
+        self.script_canvas_components[0].set_component_property_value(SCRIPT_CANVAS_COMPONENT_PROPERTY_PATH, sourcehandle)
+
+    def set_component_graph_file(self, sc_file_path: str, component_index=0) -> tuple:
```

Comment:
`set_component_graph_file_from_path`

Reply:
updated the function name and changed what it returns since we don't need the old data anymore.